folioreader/src/main/java/com/folioreader/util/HighlightUtil.java

```diff
@@ -30,13 +40,17 @@ public static String createHighlightRangy(String content, String bookTitle, Stri
             highlight.setContent(textContent);
             highlight.setType(color);
             highlight.setPageNumber(pageNo);
-            highlight.setScrollPosition(scrollPosition);
             highlight.setBookId(bookTitle);
             highlight.setPageId(pageId);
             highlight.setRangy(rangyHighlightElement);
             highlight.setDate(Calendar.getInstance().getTime());
             // save highlight to database
-            HighLightTable.insertHighlight(highlight);
+            long id = HighLightTable.insertHighlight(highlight);
+            if (id != -1) {
+                highlight.setId((int) id);
+                LocalBroadcastManager.getInstance(context).sendBroadcast(
```

Comment:
can we create method sendHighlightBroadcastEvent() method in HighlightUtil? which will use this statement?